solutions/swb-app/src/dataSetRoutes.ts

```diff
@@ -43,18 +43,16 @@ import { validateAndParse } from './validatorHelper';
 export function setUpDSRoutes(router: Router, dataSetService: DataSetPlugin): void {
   // creates new prefix in S3 (assumes S3 bucket exist already)
   router.post(
-    '/datasets',
+    '/projects/:projectId/datasets',
     wrapAsync(async (req: Request, res: Response) => {
       const validatedRequest = validateAndParse<CreateDataSetRequest>(CreateDataSetRequestParser, req.body);
-      const dataSet = await dataSetService.provisionDataSet({
+      const dataSet = await dataSetService.provisionDataSet(req.params.projectId, {
```

Comment:
where do you validate projectId?

Reply:
Not sure if @aws-tyler will be adding that in with his dataset merge, but we do have tasks to add in input validation after this.

Reply:
Also, authz layer prevents this from being called with an invalid projectId

Reply:
Dataset changes will be removed from this PR; this branch was tracking updates that were being made, but the PR is not expecting to merge the final changes to datasets into `swb/project-as-a-boundary`. A separate PR will be made to add all dataset changes in.